.pullapprove.yml

```
@@ -754,7 +1170,40 @@ groups:
       approved: "comp: testing"
       rejected: "comp: testing"
     reviewers:
-      users:
+      users: # Keep list in sync with Docs: Testing
+        - AndrewKushnir
+        - IgorMinar
+        # OOO as of 2020-09-28 - pkozlowski-opensource
+
+
+  # =========================================================
+  #  Docs: Testing
+  # =========================================================
+  docs-testing:
+    <<: *defaults
+    conditions:
+      - *can-be-global-approved
+      - *can-be-global-docs-approved
+      - >
+        contains_any_globs(files.exclude('packages/compiler-cli/**'), [
```

The `exclude` is not necessary here:

```suggestion
        contains_any_globs(files, [
```

---

Again, thanks for catching!